ingestion/src/metadata/ingestion/source/database/databricks/usage.py

```diff
@@ -102,6 +102,9 @@ def process_table_query(self) -> Optional[Iterable[TableQuery]]:
                                 endTime=row.get("execution_end_time_ms"),
                                 analysisDate=datetime.now(),
                                 serviceName=self.config.serviceName,
+                                duration=row.get("duration") / 1000
+                                if row.get("duration")
+                                else None,
```

Comment:
will it be possible to edit the sqlalchemy query and get it divided by 1000 there itself

Reply:
for databricks we are not executing a query to fetch query history instead we are hitting a rest api